File: server/src/main/java/com/otl/server/dto/ScheduleDto.java
package com.otl.server.dto;

import java.time.LocalDateTime;

import com.otl.server.model.Schedule;
import lombok.Data;


@Data
public class ScheduleDto {
	private String homeTeamName;
	private String awayTeamName;
	private Integer homeTeamScore;
	private Integer awayTeamScore;
	private Integer channelCode;
	private String gameDate;
	private String gameTime;

	public ScheduleDto(Schedule schedule) {
		this.homeTeamName = schedule.getHomeTeamName();
		this.awayTeamName = schedule.getAwayTeamName();
		this.homeTeamScore = schedule.getHomeTeamScore();
		this.awayTeamScore = schedule.getAwayTeamScore();
		this.channelCode = schedule.getChannelCode();
		LocalDateTime localDateTime = schedule.getGameDateTime();
		this.gameDate = String.format("%d.%d", localDateTime.getMonthValue(), localDateTime.getDayOfMonth());
		this.gameTime = String.format("%d:%d", localDateTime.getHour(), localDateTime.getMinute());
	}
}

File: server/src/main/java/com/otl/server/dto/ScheduleDto.kt
package com.otl.server.dto

import com.otl.server.model.Schedule

data class ScheduleDto(var schedule: Schedule) {
	val homeTeamName: String = schedule.homeTeamName
	val awayTeamName: String = schedule.awayTeamName
	val homeTeamScore: Int = schedule.homeTeamScore
	val awayTeamScore: Int = schedule.awayTeamScore
	val channelCode: Int = schedule.channelCode
	val gameDate: String
	val gameTime: String

	init {
		val localDateTime = schedule.gameDateTime
		gameDate = String.format("%d.%d", localDateTime.monthValue, localDateTime.dayOfMonth)
		gameTime = String.format("%d:%d", localDateTime.hour, localDateTime.minute)
	}
}